### DISCUSSION TITLE
Error：if any <PackageReference> exists in the <ItemGroup> section of the [.csproj] file

### DISCUSSION THREAD

**Discussion by jack-liew** [dotnet/dotnet-docker#4238] [Q&A]
Hi:
   if any `<PackageReference>` exists in the `<ItemGroup>` section of the .csproj file, When dockerfile is run to [RUN dotnet restore], the error is as follows:
```sh
Step 4/10 : RUN dotnet restore
 ---> Running in ad1bb62b35a4
  Determining projects to restore...
/source/aspnetapp.csproj : error NU1301: Unable to load the service index for source https://api.nuget.org/v3/index.json.
  Failed to restore /source/aspnetapp.csproj (in 5.89 sec).
The command '/bin/sh -c dotnet restore' returned a non-zero code: 1
```
My a.csproj:
```xml
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>net7.0</TargetFramework>
    <Nullable>enable</Nullable>
    <ImplicitUsings>enable</ImplicitUsings>
    <UserSecretsId>57393389627611478466</UserSecretsId>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
  </ItemGroup>

</Project>
```
</Project>
```

Thanks~~~

**Comment by mthalman** [comment]
This is most likely due to the configuration of Docker on the machine being unable to communicate with https://api.nuget.org, through a proxy perhaps.

Example questions on this topic:
https://stackoverflow.com/questions/57824669/nuget-with-docket-unable-to-load-the-service-index-for-source
https://stackoverflow.com/questions/50347517/docker-unable-to-load-the-service-index-for-source-https-api-nuget-org-v3-inde
